### DISCUSSION TITLE
Payment option other than paypal and google play store

### DISCUSSION THREAD

**Discussion by Mohammad78SD** [tribalfs/GalaxyMaxHzPub#233] [Q&A] [answered]
Hi there developer thanks for your amazing app, I have s21+ and its battery life is killing me, i have to use psm to get through a day but i love how 120hz feels and looks. Today i found this app in youtube and installed it and its working fine for me but i noticed that it is a trial and i have to purchase a license for my device but sadly in my country (iran) our banks and cards doesnt support payment to other countries and i cant pay for the premium license. Is there another way for me to use this amazing app?

**Comment by Xisrr1** [comment]
You didn't get an answer, which means the developer doesn't want to help. 

I stopped using this app ever since I saw the app is using the internet suspiciously.

**Reply by tribalfs** [reply, marked as answer]
> You didn't get an answer, which means the developer doesn't want to help.


Sorry, but we can't help because we can't provide any other option.

> I stopped using this app ever since I saw the app is using the internet suspiciously.


That strange. It's only using the internet for checking updates, for activation check and when performing "reload profiles".  Anyway, if you have doubts, you can always submit the app to any virus scanner.
